docs/standard/assemblies-and-the-global-assembly-cache.md

```diff
@@ -0,0 +1,56 @@
+  
+## Assembly Manifest  
+ Within every assembly is an *assembly manifest*. Similar to a table of contents, the assembly manifest contains the following:  
+  
+-   The assembly's identity (its name and version).  
+  
+-   A file table describing all the other files that make up the assembly, for example, any other assemblies you created that your .exe or .dll file relies on, or even bitmap or Readme files.  
+  
+-   An *assembly reference list*, which is a list of all external dependencies—.dlls or other files your application needs that may have been created by someone else. Assembly references contain references to both global and private objects. Global objects reside in the global assembly cache, an area available to other applications, somewhat like the System32 directory. The <xref:Microsoft.VisualBasic?displayProperty=nameWithType> namespace is an example of an assembly in the global assembly cache. Private objects must be in a directory at either the same level as or below the directory in which your application is installed.  
+  
+ Because assemblies contain information about content, versioning, and dependencies, the applications you create with C# or Visual Basic do not rely on Windows registry values to function properly. Assemblies reduce .dll conflicts and make your applications more reliable and easier to deploy. In many cases, you can install a .NET-based application simply by copying its files to the target computer.  
+  
+ For more information see [Assembly Manifest](../../../../../docs/framework/app-domains/assembly-manifest.md).  
+  
+## Adding a Reference to an Assembly  
+ To use an assembly, you must add a reference to it. Next, you can use the [using directive](../../../../csharp/language-reference/keywords/using-directive.md) for C# or [Imports statement](../../../../visual-basic/language-reference/statements/imports-statement-net-namespace-and-type.md) for Visual Basic to choose the namespace of the items you want to use. Once an assembly is referenced and imported, all the accessible classes, properties, methods, and other members of its namespaces are available to your application as if their code were part of your source file.  
+  
+ In C#, you can also use two versions of the same assembly in a single application. For more information, see [extern alias](../../../../csharp/language-reference/keywords/extern-alias.md).  
+  
+## Creating an Assembly  
+ Compile your application by building it from the command line using the command-line compiler. For details about building assemblies from the command line, see [Command-line Building With csc.exe](../../../../csharp/language-reference/compiler-options/command-line-building-with-csc-exe.md) for C# or see [Building from the Command Line](../../../../visual-basic/reference/command-line-compiler/building-from-the-command-line.md) for Visual Basic.  
+  
+> [!NOTE]
+>  To build an assembly in Visual Studio, on the **Build** menu choose **Build**.  
+  
```

Comment:
Along with the See also section, add a section here:
\## In this section
\[.NET assembly file format](assembly-format.md)